data/multiplayer/README.multiplayer

```diff
@@ -1,24 +1,23 @@
-----------------------------------------------------------------------
-                   Multiplayer Freeciv Ruleset
-----------------------------------------------------------------------
+Multiplayer Freeciv21 Ruleset
+-----------------------------
 
-The 'multiplayer' ruleset is intended to rectify some game balance
+The "multiplayer" ruleset is intended to rectify some game balance
 issues encountered on multiplayer servers with the classic ruleset.
 It is based on the 'rewonder' ruleset designed by Madeline Book for
-Freeciv 2.1.
+legacy Freeciv 2.1.
 
 This ruleset is also suited for Longturn and Play-By-Email games.
 
 
-1. Changes compared to the classic ruleset
-   ---------------------------------------
+Changes Compared to the Classic Ruleset
+=======================================
 
 The differences can be extracted by
 
-#> diff -ur data/classic data/multiplayer
+$ diff -ur data/classic data/multiplayer
```

Comment:
Does this get rendered as fixed-width text?

Reply:
Nope, neither does the `#>`. that is why I went with the `$` make it look more like a command prompt.